launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -93,7 +93,7 @@
    * Maximum time (in ms) to wait for a child process to connect back to the launcher server
    * when using @link{#start()}.
    */
-  public static final String CHILD_CONNECTION_TIMEOUT = "spark.launcher.childConectionTimeout";
+  public static final String CHILD_CONNECTION_TIMEOUT = "spark.launcher.childConnectionTimeout";
```

Comment:
Please see: https://github.com/apache/spark/pull/30323#discussion_r521449342

Reply:
link: https://github.com/apache/spark/pull/30679

Reply:
Hmmmm OK I will close this one.